x-pack/plugins/maps/public/shared/layers/sources/es_source.js

```diff
@@ -270,6 +283,24 @@ export class AbstractESSource extends AbstractVectorSource {
     }
   }
 
+  async createShapeFilter(geojsonPolygon) {
```

Comment:
Looks like this is duplicated function from x-pack/plugins/maps/public/index_pattern_util.js